logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/call/GlobalCallManager.kt

```diff
@@ -32,6 +32,7 @@ import com.wire.kalium.logic.data.id.CurrentClientIdProvider
 import com.wire.kalium.logic.feature.call.usecase.ConversationClientsInCallUpdater
 import com.wire.kalium.logic.feature.message.MessageSender
 import com.wire.kalium.logic.featureFlags.KaliumConfigs
+import com.wire.kalium.network.NetworkStateObserver
```

Comment:
I think the failing test is because of this line, we are accessing something inside the feature package (we should have here I think only use cases), We might try to solve it by moving this class to the repository package, since is not needed for AR ? 🤔

Reply:
Yeah but I would preferto not create a data package for each platform, I will just ignore this rule for those classes